core/src/main/java/org/elasticsearch/action/search/SearchResponse.java

```diff
@@ -240,4 +330,87 @@ public void writeTo(StreamOutput out) throws IOException {
     public String toString() {
         return Strings.toString(this);
     }
+
+    public static class InternalSearchResponse {
```

Comment:
ok because although the class can be created separately, it should always be used as part of the SearchResponse when serializing?